app/views/wiki/new.html.erb

```diff
@@ -40,6 +40,10 @@ See COPYRIGHT and LICENSE files for more details.
 
 <div id="preview"></div>
 
+<op-sidemenu
+  data-items='[{ "title": "root", "href": "/" }]'
+></op-sidemenu>
```

Comment:
Example usage. Data has to be JSON serialized

Reply:
To actually see this on the new wiki page, you have to hover it since the text is white by default :P

Reply:
To avoid JSON parsing errors, I'd prefer if we conventialize it like this:

```
<%= content_tag :op_sidemenu,
                             data: {
                             items: { title: 'root', href: 'foo }.to_json
                             }
```